src/demos/Buttons.vue

```diff
@@ -389,10 +280,10 @@ components: {
         <p>Used for Download Streamlabs OBS buttons.</p>
         <DemoSection title="Download" :code="demoCode">
           <template #components>
```

Comment:
Can you wrap these buttons with this `<div class="s-button-container s-button-container--left">{{ BUTTONS CODE }}</div>`. That way the are sitting next to each other.